src/test/java/spoon/test/constructorcallnewclass/NewClassTest.java

```diff
@@ -157,7 +157,7 @@ public void testCtNewClassInNoClasspath() throws Exception {
 		final CtClass anonymousClass = ctNewClass.getAnonymousClass();
 		assertNotNull(anonymousClass);
 		assertNotNull(anonymousClass.getSuperclass());
-		assertEquals("Lock.With", anonymousClass.getSuperclass().getSimpleName());
```

Comment:
add an assertion on getSimpleName() as well? (I guess it should be "With" or "Lock")